go.mod

```diff
@@ -29,9 +29,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
-// Until controller-runtime releases with v0.26 support
```

Comment:
Please update the commit message to match what this commit is doing now (dropping the replace, not upgrading to controller-runtime 0.14.0).